src/main/java/com/load/service/loadServiceImpl.java

```
@@ -0,0 +1,169 @@
+		load.setUnloadingPoint(payLoadRequest.getUnloadingPoint());
+		load.setNoOfTrucks(payLoadRequest.getNoOfTrucks());
+		load.setTruckType(payLoadRequest.getTruckType());
+		load.setProductType(payLoadRequest.getProductType());
+		load.setWeight(payLoadRequest.getWeight());
+		load.setDate(payLoadRequest.getDate());
+		load.setShipperId("shipper:"+loadid);
+		if(payLoadRequest.getComment()!=null) {
+			load.setComment(payLoadRequest.getComment());
+		}
+		
+		load.setStatus(Constants.pending);
+		loadResponse.setStatus(Constants.pending);
+		payloadRepository.save(load);	
+		return loadResponse;
+		
+		
+	}
+
+
+	@Override
+	public PayloadResponse updatePayload(String id,PayloadRequest payLoadRequest) {
+		// TODO Auto-generated method stub
+		PayloadResponse loadResponse = new PayloadResponse();
+		Payload load = new Payload();
+
+		Optional<Payload> L = payloadRepository.findById(id);
+		if(L.isPresent()) {
+			load = L.get();
+		}
+		else {
+			loadResponse.setStatus(Constants.AccountNotFoundError);
+			return loadResponse;
+		}
+
+		
+		if(payLoadRequest.getLoadingPoint()!=null) {
+			load.setLoadingPoint(payLoadRequest.getLoadingPoint());
+		}
+		if(payLoadRequest.getUnloadingPoint()!=null) {
+			load.setUnloadingPoint(payLoadRequest.getUnloadingPoint());
+		}
+		if(payLoadRequest.getProductType()!=null) {
+			load.setProductType(payLoadRequest.getProductType());
+		}
+		if(payLoadRequest.getTruckType()!=null) {
+			load.setTruckType(payLoadRequest.getTruckType());
+		}
+		if(payLoadRequest.getNoOfTrucks()!=0) {
+			load.setNoOfTrucks(payLoadRequest.getNoOfTrucks());
+		}
+		if(payLoadRequest.getWeight()!=0.0) {
+			load.setWeight(payLoadRequest.getWeight());
+		}
+		if(payLoadRequest.getComment()!=null) {
+			load.setComment(payLoadRequest.getComment());
+		}
+		if(payLoadRequest.getStatus()!=null) {
+			load.setStatus(payLoadRequest.getStatus());
+		}
+		if(payLoadRequest.getDate()!=null) {
+			load.setDate(payLoadRequest.getDate());
+		}
+
```

if shipperId is not null, you can return a error message as it cant be updated.